content/en/blog/2020/istiocon-2021/istiocon-2021.md

```diff
@@ -0,0 +1,16 @@
+# Join us for the first IstioCon in 2021!
```

Comment:
You need to format using standard headers like all Istio blogs.
```suggestion
---
title: "Join us for the first IstioCon in 2021!"
description: The inaugural conference for Istio will take place at the end of February.
publishdate: 2020-12-03
attribution: "Istio Steering Committee"
keywords: [istioCon,Istio,conference]
---
```

Reply:
Also, you need to rename this file from `istiocon-2021/istiocon-2021.md` -> `istiocon-2021/index.md`.

Reply:
added and changed!